double-check-tmp.md

```diff
@@ -0,0 +1,28 @@
+- No copyright/license mentioned
+  + http://www.jjj.de/fxt/fxtbook.pdf
+  + http://igm.univ-mlv.fr/~mac/REC/text-algorithms.pdf
+  + http://www.ethoberon.ethz.ch/WirthPubl/CBEAll.pdf
+  + http://lampwww.epfl.ch/~schinz/thesis-final-A4.pdf
+  + http://www.stack.nl/~marcov/compiler.pdf
+  + http://arxiv.org/pdf/1206.1754v2.pdf
+  + http://www.csee.umbc.edu/csee/research/cadip/readings/IR.report.120600.book.pdf
```

Comment:
Author (was) an employee of the US Government (Fort Meade). It is widely cited as DOD Technical Report TR-R52-008-001 which would confirm its status as a public domain document. OK to link.